### PR TITLE
Add dhcpv6 dictionary.rfc6355

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

File: share/dictionary/dhcpv6/dictionary.rfc6355 [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+##############################################################################
+#
+#  RFC 6355 - Definition of the UUID-Based DHCPv6 Unique Identifier (DUID-UUID)
+#
+#  $Id$
+#
+##############################################################################
+
+ATTRIBUTE	DUID-Type			3	uint8
+VALUE	DUID-Type		LLT			1
```
Comment (arr2036) [inline share/dictionary/dhcpv6/dictionary.rfc6355]: These should match the spec with the `DUID-` prefix

File: share/dictionary/dhcpv6/dictionary.rfc6355 [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+VALUE	DUID-Type		EN			2
+VALUE	DUID-Type		LL			3
+
+ATTRIBUTE	DUID-UUID		4	octets
```
Comment (arr2036) [inline share/dictionary/dhcpv6/dictionary.rfc6355]: This is always 128bits so should be defined as a fixed width octets attribute

File: share/dictionary/dhcpv6/dictionary.rfc6355 (hunk 1 above)
Comment (jpereira) [reply share/dictionary/dhcpv6/dictionary.rfc6355]: done

File: share/dictionary/dhcpv6/dictionary.rfc6355 (hunk 2 above)
Comment (jpereira) [reply share/dictionary/dhcpv6/dictionary.rfc6355]: done

File: share/dictionary/dhcpv6/dictionary.rfc6355 [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+#  $Id$
+#
+##############################################################################
+
+ATTRIBUTE	DUID-Type			3	uint8
+VALUE	DUID-Type		DUID-LLT			1
+VALUE	DUID-Type		DUID-EN			2
+VALUE	DUID-Type		DUID-LL			3
+
+ATTRIBUTE	DUID-UUID		4	octets[128]
```
[the remark this answers is not in the document]
Comment (arr2036) [reply share/dictionary/dhcpv6/dictionary.rfc6355]: Should be 16

**Pull request merged**